### PR TITLE
Dbeaver/pro#2461 add support for create and alter tableview statements

### PR DESCRIPTION
**Pull request by E1izabeth**
CREATE TABLE and ALTER TABLE statements
- get highlighting of columns and tables
- have warnings
image
- support navigation with ctrl+click inside query or to properties editor
- have outline support
image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/model/ddl/SQLQueryTableConstraintKind.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.sql.semantics.model.ddl;
+
+public enum SQLQueryTableConstraintKind {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/model/ddl/SQLQueryTableConstraintKind.java:19]: Why not DBSEntityConstraintType?

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/model/ddl/SQLQueryColumnConstraintKind.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.sql.semantics.model.ddl;
+
+public enum SQLQueryColumnConstraintKind {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/model/ddl/SQLQueryColumnConstraintKind.java:19]: Why not DBSEntityConstraintType?

**Comment by uslss** [comment]
Alter table query for rename column isn't shown in outline for some reason
`ALTER TABLE public.mushrooms RENAME COLUMN toxicity_id TO toxicity_id1;`
image

**Comment by uslss** [comment]
verified

**Pull request merged**